packages/firestore/src/core/types.ts

```diff
@@ -49,7 +54,7 @@ export enum OnlineState {
   /**
    * The client considers itself offline. It is either trying to establish a
    * connection but failing, or it has been explicitly marked offline via a call
-   * to disableNetwork().
+   * to disableNetwork(). Events will be raised from cache.
```

Comment:
Per my previous comments, it's more nuanced than this so we should either drop this or rephrase it somehow (like maybe "Higher-level components should operate in offline mode.", but that's not that different than "The client considers itself offline.")

Reply:
I rephrased this a little, let me know if you think it's correct now.